pkg/rewards/7_goldActiveODRewards.go

```diff
@@ -0,0 +1,154 @@
+package rewards
+
+import (
+	"database/sql"
+
+	"github.com/Layr-Labs/sidecar/pkg/rewardsUtils"
+	"go.uber.org/zap"
+)
+
+var _7_goldActiveODRewardsQuery = `
+create table {{.destTableName}} as
+WITH od_rewards_with_block_info as (
+		select
```

Comment:
can we tidy up the formatting for this entire query? Looks like it was pasted but not formatted properly

Reply:
Hmmm. Interesting it actually looks formatted on my text editor.

Reply:
Done in [b3f5ebe](https://github.com/Layr-Labs/sidecar/pull/131/commits/b3f5ebe465a23b609b3b3e804a45148361c7af47)